packages/bigcommerce-payments-utils/src/create-bigcommerce-payments-fastlane-utils.ts

```diff
@@ -1,7 +1,5 @@
-import { BrowserStorage } from '@bigcommerce/checkout-sdk/storage';
-
 import BigCommercePaymentsFastlaneUtils from './bigcommerce-payments-fastlane-utils';
 
 export default function createBigCommercePaymentsFastlaneUtils(): BigCommercePaymentsFastlaneUtils {
-    return new BigCommercePaymentsFastlaneUtils(new BrowserStorage('paypalFastlane'));
+    return new BigCommercePaymentsFastlaneUtils();
```

Comment:
Suggestion

Maybe would be better to provide cookie name and set it as a private variable to use whole class?